lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -422,6 +428,14 @@ def add_instalments_data(xml, options)
         end
       end
 
+      def add_dynamic_interaction_type(xml, options)
+        xml.tag! 'dynamicInteractionType', 'type' => options[:dynamic_interaction_type]
+      end
+
+      def add_exemption(xml, options)
+        xml.tag! 'exemption', 'type' => options[:exemption], 'placement'=>'AUTHORISATION'
```

Comment:
I had something similar, but I think `placement` can be a lot of different things: https://github.com/activemerchant/active_merchant/compare/worldpay_3ds2?expand=1

Reply:
ah, nice! maybe I'll just remove the exemption field from this PR since you've got it covered there with the other 3ds2 additions, and just do MOTO in this PR